dev_tools/requirements/envs/dev.env.txt

```diff
@@ -58,10 +60,13 @@ iniconfig==2.0.0
 isort==6.0.0
     # via pylint
 jax==0.4.38
-    # via -r deps/resource_estimates_runtime.txt
+    # via
+    #   -r deps/resource_estimates_runtime.txt
+    #   -r deps/resource_estimates_runtime.txt
```

Comment:
Why are all these comments repeated?

Reply:
> Why are all these comments repeated?

Not sure, since this was auto generated. I can delete them. Another commit coming up in a moment.